src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.iOS.cs

```diff
@@ -1102,12 +1169,11 @@ public void RefreshLayout()
 		/// Use this method instead of NSIndexPath.FromRowSection, as the interop call
 		/// is quite costly.
 		/// </remarks>
-		protected NSIndexPath GetNSIndexPathFromRowSection(int row, int section)
+		protected NSIndexPath? GetNSIndexPathFromRowSection(int row, int section)
```

Comment:
Making `NSIndexPath` nullable here changes a few other flows in this file.

Reply:
@ajpinedam is this still relevant?

Reply:
Not at all, was just a comment for the reviewer (@jeromelaban )